ui/packages/catalog-ui-search/src/main/webapp/component/singletons/session-auto-renew.js

```diff
@@ -35,7 +35,7 @@ var sessionAutoRenewModel = new (Backbone.Model.extend({
     handleExpiryTimeResponse: function(response) {
         var msUntilTimeout = parseInt(response);
         var msUntilAutoRenew = Math.max(msUntilTimeout * 0.7, msUntilTimeout - 60000);  // 70% or at least one minute before
-        this.set('sessionRenewDate', Date.now() + msUntilAutoRenew);
+        this.set('sessionRenewDate', Date.now() + Math.trunc(msUntilAutoRenew));
```

Comment:
Why are we truncating now?

Reply:
I didn't see any documentation on how setTimeout handles fractional milliseconds, so I did this so it'd be consistent. I'll revert it if you'd prefer.